include/kframework/foundry.md

```diff
@@ -884,13 +884,29 @@ function stopPrank() external;
 
 Gas Manipulation
 ----------------
+### `setGas` - Sets the current gas left (reported by `GAS` opcode) to a specific amount.
+
+```
+function setGas(uint256 newGas) external;
+```
+
+`setGas` is useful when writing tests that depend on the gas used, and so a specific concrete amount is needed instead of the default infinite gas.
+
+```{.k .bytes}
+    rule [foundry.call.setGas]:
+         <k> #call_foundry SELECTOR ARGS => . ... </k>
+         <gas> _ => #asWord(ARGS) </gas>
+         <callGas> _ => 0 </callGas>
+      requires SELECTOR ==Int selector ( "setGas(uint256)" )
+```
+
+### `infiniteGas` - Sets the remaining gas to an infinite value.
 
 ```
 function infiniteGas() external;
 ```
 
-Set the remaining gas to an infinite value.
-This is useful for running tests without them running out of gas.
+`infiniteGas` is useful for running tests without them running out of gas. It is applied by default.
```

Comment:
Nice changes! One tidbit, we're trying to maintain a format of having one sentence per line in the markdown files.

Reply:
Thank you! Right, sorry, I've split these two sentences into separate lines in https://github.com/runtimeverification/evm-semantics/commit/b4335416a272d175c8a811e79ff0fc7dc430338e.